src/codeu/chat/common/SentimentScore.java

```diff
@@ -0,0 +1,117 @@
+package codeu.chat.common;
+
+import codeu.chat.util.Serializer;
+import codeu.chat.util.Serializers;
+import com.google.cloud.language.spi.v1beta2.LanguageServiceClient;
+
+import com.google.cloud.language.v1beta2.AnalyzeSentimentResponse;
+import com.google.cloud.language.v1beta2.Document;
+import com.google.cloud.language.v1beta2.Document.Type;
+import com.google.cloud.language.v1beta2.Sentiment;
+import java.io.IOException;
+import java.io.InputStream;
+import java.io.OutputStream;
+
+/**
+ * Created by rsharif on 5/3/17.
+ */
+public class SentimentScore {
+
+  private int score;              // the average score for this object
+  private int numScores;          // this will mark how many messages have been included in the calculation of the current score
+
+  // The language client is used to connect to the natural language API.
+  // todo: requires environment variable GOOGLE_APPLICATION_CREDENTIALS=<PATH/TO/SERVICE-ACCOUNT/JSON>
+  /*
+   * Issue - How will this work with the google hosting. How will we make the environment variable
+   * and point to the json file. Will this be feasible?
```

Comment:
I actually wrote in this comment before I fixed up the shell files. I just submitted a separate issue describing this issue in more depth... #18